web/src/styles/global.css

```
@@ -162,6 +162,17 @@ body {
     transform: translateY(0);
   }
 }
+
+@keyframes fadeOut {
```

Stylelint (17.10.0)

[error] Expected keyframe name "fadeOut" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)